test/functional/ui/inccommand_spec.lua

```diff
@@ -376,7 +376,7 @@ describe(":substitute, 'inccommand' preserves undo", function()
 
   -- TODO(vim): This does not work, even in Vim.
   -- Waiting for fix (perhaps from upstream).
-  pending("at a non-leaf of the undo tree", function()
+  it("at a non-leaf of the undo tree", function()
```

Comment:
While we're in the area I think it's ok to remove these lines from `inccommand_spec.lua`:

     if helpers.pending_win32(pending) then return end

the tests should work on Windows.